lib/locale.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'singleton'
+
+class Locale
```

Comment:
I think will be better to use `I18n` for naming

Reply:
And why not using `i18n` itself?